gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { Visibility } from '@gravitee/management-webclient-sdk/src/lib/models';
+import faker from '@faker-js/faker';
+import apiDetails from '@pageobjects/Apis/ApiDetails';
+import { ApiImport } from '@model/api-imports';
+
+describe('API Info Page functionality', () => {
+  beforeEach(() => {
+    cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+    cy.visit('/#!/environments/default/apis/');
+    cy.url().should('include', '/apis/');
+  });
+
+  before(() => {
+    cy.log('Import (create) v2 API');
+    cy.request({
+      method: 'POST',
+      url: `${Cypress.env('managementApi')}${Cypress.env('defaultOrgEnv')}/apis/import`,
+      auth: { username: ADMIN_USER.username, password: ADMIN_USER.password },
+      body: ApisFaker.apiImport({ visibility: Visibility.PUBLIC }),
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      api = response.body;
+    });
+  });
+
+  const ApiDetails = new apiDetails();
+  const apiName = faker.commerce.productName();
+  const newName = apiName.replace(/ /g, '-');
+  const apiVersion = faker.datatype.number();
+  const apiDescription = faker.commerce.productDescription();
```

Comment:
You should avoid using `faker.commerce.productDescription()` for the description of an API has a recent PR sanitize the description of an API on backend side. It can cause a different encoding when doing the assertion, and so flaky tests
See this commit: https://github.com/gravitee-io/gravitee-api-management/commit/9c6e88efdd1204709962d7785053a26e2dbe8f21

Reply:
Have changed this to lorem.words(10) , but would like to know why/where it's not able to deal with certain special characters as would expect users to use a special character in that field every now and then?

Reply:
see the jira linked to this PR: https://github.com/gravitee-io/gravitee-api-management/pull/4713